pkg/queue/manager.go

```diff
@@ -48,17 +53,59 @@ type Manager struct {
 	clusterQueues map[string]ClusterQueue
 	localQueues   map[string]*LocalQueue
 
+	wm        sync.RWMutex
```

Comment:
why `wm`? Maybe `sm` for Snapshots Mutex? Or just explicitely `snapshotsMutex`.

Reply:
renamed